### ISSUE TITLE
Import missing codes for @numeric variables

### ISSUE THREAD

**Issue by tmelliott** [iNZightVIT/iNZightTools#174]
e.g., 
```
#' @numeric y na=888,999
```
At first, just read 888 and 999 as NA, but in future maybe create new column `y_missing` if reason codes are provided, e.g.,
```
#' @numeric y na[Refuse=888,DontKnow=999]
```

**Comment by tmelliott** [comment]
First bit completed with a9b6e0c000

**Comment by tmelliott** [comment]
Closed with bb1375aa
